### DISCUSSION TITLE
help getting 0.23.2 on Android Firefox nightly

### DISCUSSION THREAD

**Discussion by taracdia** [AprilSylph/XKit-Rewritten#1343] [Help Desk] [answered]
Apologies if the answer is simply "you can't do that yet." I'm trying to get the version 0.23.2 on my Firefox nightly for Android add on. I'm trying to get rid of the horrible recommended blogs thing. I saw that a fix has been made and released in version 0.23.2 but even though i uninstalled and reinstalled the addon I'm still seeing the recommended blogs and my version is 0.23.1. im not sure whether I'm doing something wrong or if it's just not available yet so i thought i should ask just in case

**Comment by marcustyphoon** [comment, marked as answer]
You are indeed correct: you can't get it until Mozilla releases it! I'll try to remember to ping you when that's the case.

**Reply by taracdia** [reply]
That would be fantastic! Thank you so much for your quick reply!

**Reply by marcustyphoon** [reply]
@taracdia 0.23.2 should be available now, though it doesn't have a fix for #1346!

**Reply by taracdia** [reply]
Awesome thank you!
